### PR TITLE
Create ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by mmed-c8y

Ticket: [MTM-56510](https://cumulocity.atlassian.net/browse/MTM-56510)
Original PR: [7309](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7309)

[MTM-56510]: https://cumulocity.atlassian.net/browse/MTM-56510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: cherry-pick of all feature: 31cccab4f43f2bcc51a132dcb402b1a… (#7309)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: The Claude AI suggested to use the following title:
```suggestion
title: All feature cherry-picked from main branch
```

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+cherry-pick of all feature: 31cccab4f43f2bcc51a132dcb402b1a… (#7309)
```
Comment (gen-App-Enablement) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: The Claude AI suggested to use the following body:
```suggestion
As part of ongoing development efforts, a commit containing an "all feature" was made on the main branch. To ensure this feature is also available on the current branch, the commit has been cherry-picked and merged. This change brings the current branch up-to-date with the latest "all feature" from the main branch. End-users should not experience any noticeable impact, as this is an internal development change.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md (hunk 2 above)
Comment (kowa-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md:17]: @mmed-c8y please verify the suggestions as current release note is to technical

**Review by kowa-c8y — CHANGES_REQUESTED** [review]

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+When deleting or modifying a global role, the administrator has the option to log out all users assigned to that role.
```
Comment (kowa-c8y) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
When deleting or modifying a global role, the administrator has the option to log out all users who are assigned to that role.
```

**Review by kowa-c8y — APPROVED** [review]

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Logging out all users when deleting/changing global roles.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
title: Logging out all users when deleting or modifying a global role
```

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md (hunk 3 above)
Comment (BeateRixen) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: I suggest to add "now": the administrator now has the option...

**Comment by BeateRixen** [comment]
@kowa-c8y @mmed-c8y @saran-govindarajan Can we get this PR merged? It already quite old and change logs should be added quickly (latest within a week) to get published in time.

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+When deleting or modifying a global role, the administrator has the option to log out all users who are assigned to that role.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
When deleting or modifying a global role, the administrator has the option to log out all users that are assigned to that role.
```

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+When deleting or modifying a global role, the administrator has the option to log out all users that are assigned to that role.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.
```

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: @saran-govindarajan @JanePorter This information is quite poor. What was the issue the user was facing and what is the advantage of this option? We need more details here to make this entry of any use to the user. This PR is now open for a couple of weeks, actually change log PR should be merged asap.

Comment (kowa-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: Actually, it seems that the description is complete - previously there was no option to log all users out when modifying a global role, but now there is. Optionally, we can add information about automatic refresh of roles when a relogin is forced.

**Comment by kowa-c8y** [comment]
From my point of view it can be merged. I already approved it. If @BeateRixen  will confirm it it will be automatically, merged.

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Logging out all users when deleting or modifying a global role
```
Comment (BeateRixen) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
title: Option to log out all users when deleting or modifying a global role
```

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md (hunk 7 above)
Comment (JanePorter) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: @kowa-c8y but why would someone want to lot out all users? Should it read something like:

When deleting or modifying a global role an administrator can now ensure the changes take immediate effect by using a new option which will log out all users assigned to that role.

Comment (kowa-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: Good point! The intention here is definitely worth mentioning.  :+1:

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56510
+version: 1020.36.0
+---
+When deleting or modifying a global role, the administrator now has the option to log out all users that are assigned to that role.
```
Comment (kowa-c8y) [inline content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]:
```suggestion
When deleting or modifying a global role, an administrator can now ensure that changes take immediate effect by using a new option to log out all users assigned to that role.
```

Comment (kowa-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: @BeateRixen please check if it's ok commit and we will merge it :+1:

File: content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md (hunk 7 above)
Comment (kowa-c8y) [reply content/change-logs/platform-services/ui-c8y-1020-36-0-cherry-pick-of-all-feature-31-cccab-4-f-43-f.md]: @BeateRixen  please check last commit and if its ok please merge it

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**